.prettierignore

```diff
@@ -0,0 +1,14 @@
+**/*.graphql
```

Comment:
remove such stuff that is not present in your env, such as generated and dynamodb and grapqhl. Include only stuff that are present in this project and should be ignored from prettier checks